loader/src/main/kotlin/net/weavemc/loader/bootstrap/transformer/ClassLoaderTransformer.kt

```diff
@@ -27,33 +30,106 @@ object URLClassLoaderTransformer : SafeTransformer {
         reader.accept(node, 0)
 
         node.interfaces.add(internalNameOf<URLClassLoaderAccessor>())
-        node.visitMethod(Opcodes.ACC_PUBLIC, "addWeaveURL", "(Ljava/net/URL;)V", null, null).visitAsm {
+
+        // public URLClassLoader getWeaveBacking()
+        node.visitMethod(Opcodes.ACC_PUBLIC, "getWeaveBacking", "()Ljava/lang/ClassLoader;", null, null).visitAsm {
+            aload(0)
+            areturn
+        }
+
+        val defaultIgnoredPackages = listOf(
+            "net.weavemc.loader.bootstrap.",
+            "me.xtrm.klog.",
+            "kotlin."
+        )
```

Comment:
Why is delegation to system loader necessary? I imagine your library statically stores loggers for each class, but I don't see why it would be an issue that those classes get "reloaded" in a platform-specific loader? This feels quite hacky. Besides, the bootstrap code should probably use a different logger anyway, pointing to some other log file.

Reply:
Since the logger classes (most specifically the Appender implementation) is loaded on the System ClassLoader, trying to initialize a logger on the injected classloader (or the Mixin sandboxed classloader) will result in the classes getting reinitialized (but not redefined), which includes the KlogConfig global object, resetting the Appender and re-creating the file. I originally worked-around this by modifying the appender's code to detect existing log-files for the current session, but I've found this to be cleaner in terms of fix. Open to suggestions on this one.

Reply:
I think it would be a good idea to just, separate out the loggers, such that bootstrap logs go into a bootstrap-specific logfile, and then the rest of the logs would go through a different "main" logfile.

Reply:
I'd argue about bootstrap not having a whole lot to do compared to the loader. In my mind, i would give it a separate logfile if it was a separate module/process entirely. Do you still want a separation? And if so, should I add a `latest-bootstrap.log` file?

Reply:
Bump. If this isn't an issue anymore, can I ask for a re-review?